docarray/typing/tensor/embedding/embedding.py

```diff
@@ -0,0 +1,7 @@
+from typing import Union
+
+from docarray.typing.tensor.embedding.embedding_ndarray import NdArrayEmbedding
+
+# from docarray.typing.tensor.embedding.embedding_torch import TorchEmbedding
+
+Embedding = Union[NdArrayEmbedding]  # , TorchEmbedding]
```

Comment:
What's this? If torch is installed the `Embedding` should also be able to be a `TorchEmbedding`

Reply:
my bad I forgot this one